src/chapters/classes/exercises.rst

```diff
@@ -0,0 +1,13 @@
+Exercises: Classes and Objects
+==============================
+
+Work on these exercises in the IntelliJ ``java-web-dev-exercises`` project.
+
+1. Open up the file, ``StudentClass.java``, and add all of the necessary getters and setters. 
```

Comment:
I am slightly confused here.  The ``Student`` class is referenced throughout the chapter, but ``StudentClass.java`` is only introduced here.  Will this file contain the ``Student`` class, or should the file be re-titled ``Student.java``?

Reply:
Done! Thanks!